jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardDeckTests.java

```diff
@@ -0,0 +1,131 @@
+package io.github.epam.bootstrap.tests.composite.section.card;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.*;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.testng.Assert.assertEquals;
+
+/**
+ * Created by Zaur Bibilov on 27.09.2019
+ * Email: zaur_bibilov@epam.com;
+ * Skype: dasilveria
+ */
+
+public class CardDeckTests extends TestsInit {
+
+    private String card1Title = "SPIDER MAN";
+    private String card2Title = "HULK";
+    private String card1ImageSrc = "https://jdi-testing.github.io/jdi-light/images/spider-man.jpg";
+    private String card2ImageSrc = "https://jdi-testing.github.io/jdi-light/images/hulk.jpg";
+    private String card1ImageAlt = "spider-man";
+    private String card2ImageAlt = "hulk";
+    private String card1MainText = "Spider-Man is a fictional superhero created by writer-editor Stan Lee and writer-artist Steve Ditko.";
+    private String card2MainText = "The Hulk is a fictional superhero appearing in publications by the American publisher Marvel Comics.";
+    private String card1HeroName = "Peter Parker";
+    private String card2HeroName = "Bruce Banner";
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test
+    public void isCardDeckTest() {
+        cardDeckWithoutFooter.highlight();
+        cardDeckWithoutFooter.is().core().attr("class", "card-deck");
```

Comment:
better to use ready method hasClass instead of attr

Reply:
put please "card-deck" into static final variable because it's a persistent/fixed value from bootstrap framework. so if bootstrap changed it we should change only one place in our code

Reply:
Done